api/models/userDao.js

```diff
@@ -0,0 +1,57 @@
+import { dataSource } from './dataSource.js';
+
+const createUser = async (kakaoId, name, email, ageId, genderId) => {
+  return await dataSource.query(
+    `
+      INSERT INTO users(
+        kakao_id,
+        name,
+        email,
+        age_id,
+        gender_id
```

Comment:
AS를 이용해서 snake_case를 camel_case로 바꾸어서 사용해 주세요.
```suggestion
        kakao_id AS kakaoId,
        name,
        email,
        age_id,
        gender_id
```